### DISCUSSION TITLE
Sort the licenses users may pick in groups

### DISCUSSION THREAD

**Discussion by tilfischer** [NFDI4Chem/nmrxiv#965] [Ideas]
Dear all,

**Is your enhancement request related to a problem? Please describe.**

After upload of data the user is taken to another page to enter further information such as a description, keywords and many more. On the bottom of that page one needs to select a license. There is a list of many licenses down there. Without knowing too much on licenses, this list is too long and might be confusing to users. 

**Describe the solution you'd like**

The list could be grouped in 

1) licenses for research data, containing the CC licenses, Public Domain Mark 1.0, Open Data Commons Public Domain Dedication & License 1.0, Open Data Commons Attribution License v1.0, Open Data Commons Open Database License v1.0. 

and

2) licenses for software (e.g. GNU) 

and

3) others.

Best,
Tillmann

**Comment by NishaSharma14** [comment]
Currently below is the list of licenses being used in nmrXiv.
https://github.com/NFDI4Chem/nmrxiv/discussions/123

**Comment by tilfischer** [comment]
Btw: I haven't reinvented the wheel here, but also RADAR seems to group licenses. We should try to guide researchers on which license to select. For research data the CC licenses alone are sufficient, hence, it does not need to be more complicated. Moreover, CC licenses are also the way to go for research data following [forschungsdaten.info](https://forschungsdaten.info/themen/rechte-und-pflichten/forschungsdaten-veroeffentlichen/creative-commons-lizenzen/) or [HU Berlin](https://www.cms.hu-berlin.de/en/dl-en/dataman-en/share/legal-aspects/license/select-license) or [RWTH Aache](https://blog.rwth-aachen.de/forschungsdaten/en/2023/01/12/fdm-erklaert-was-ist-eigentlich-open-access/) or [TU Braunschweig](https://www.tu-braunschweig.de/en/forschung/forschungsdaten-transparenz/forschungsdaten/grundkurs-forschungsdatenmanagement/recht-im-fdm/lizenzen) or many others.
